docker/php/000-default.conf

```diff
@@ -9,8 +9,8 @@
     SetEnv REMOTE_USER ${REMOTE_USER}
 
     RewriteEngine On
-    RewriteRule (.*)/api/(.*)$ $1/api/index.php?a=$2 [QSA,L]
-    RewriteCond %{REQUEST_FILENAME}.php -f
-    RewriteRule (.*) $1.php [L]
+    RewriteCond %{REQUEST_FILENAME} !-f 
```

Comment:
What is the purpose of this Rule? and the API rewrite rule still works?

Reply:
I left this in because it was there before and didn't cause a problem, but I looked it up and it matches non existent files, which is unnecessary here. I've removed it.